### PR TITLE
Remove secret IP from putty tunnel doc

### PR DESCRIPTION
**Pull request by valju**

### CODE REVIEW

**Comment by valju** [comment]
Seems not to break anything. Good to hide secrets. (Though visible in commit history!)

File: Documentation/SSH_tunneling_guide.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ Complete either one of the following step and move on to the **Testing The Conne
 
 ### Using Putty
 
-- Host name: **195.148.22.114**
+- Host name: **SECRET_IP_WOULD_BE_HERE**
```
Comment (timoBaader) [inline Documentation/SSH_tunneling_guide.md:16]: amazing

**Comment by SharkTal** [comment]
Good, test review

**Review by timoBaader — APPROVED** [review]
5/ 10

**Pull request merged**